templates/email_list.mustache

```diff
@@ -0,0 +1,69 @@
+{{!
+    This file is part of Moodle - http://moodle.org/
+
+    Moodle is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Moodle is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+}}
+{{!
+    @template block_enrol_student/email_list
+
+    This template renders the email list.
+
+    Example context (json):
+    {
+       "users" [
+                  [
+                  "email" : s1@example.com
+                   "firstname" " student_1
+                    "lastname" : student_11
+                  ],
+                  [
+                  "email" : s2example.com
+                  "firstname" " student_2
+                  "lastname" : student_22
+                  ],
+               ]
+    }
+
+  }}
+
+<h6>Student List</h6>
+
+
+<table>
+    <thead>
+    <tr>
+        <th id="first_name_id" hidden>FirstName</th>
```

Comment:
1. Localization strings not used.
2. Why do we want to hide the table headers here?

Reply:
done